backend/schema/metadata.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: why is brand an enum instead of a string?
```

Comment:
Please remove – I addressed this question in https://github.com/datu925/electric-machines/pull/24#discussion_r1531349626. If you want to keep talking about it, feel free to start a thread in Slack or DM me.